test/node-api/test_buffer/test.js

```diff
@@ -28,4 +28,7 @@ const tick = require('util').promisify(require('../../common/tick'));
 
   // To test this doesn't crash
   binding.invalidObjectAsBuffer({});
+
+  const arrayBuffer = binding.bufferFromArrayBuffer();
+  assert(arrayBuffer instanceof ArrayBuffer, 'Expected an ArrayBuffer');
```

Comment:
Shouldn't the return value of the newly introduced function to be a `Buffer` instance?